sonic-net/sonic-buildimage
# Sync SONiC host's timezone to all containers

Sync the content of /etc/timezone and /etc/localtime from the SONiC host to all containers
during their own systemd services be started.

#### Why I did it
The containers are not awared the timezone change in the SONiC host, it may cause some info 
are inconsistent e.g. time gap between syslog and swss.rec/sairedis.rec

#### How I did it
In the systemd scripts of services, then timezone of containers will be sync in the postStartAction()
and therefore the containers will have same timezone after their own services are restarted

#### How to verify it
Change the timezone in the SONiC host and then exec the sudo config reload -y
To check the containers timezone are changed

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
